base_report_info/models/res_company.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    bank_info = fields.Char(
+        name="Bank",
+    )
```

Comment:
```suggestion
    bank_info = fields.Char()
```
It's not a normal practice to have a name attribute in fields. Where did you get the idea?

Reply:
Im sorry I miss understood between adding field to view and create field in model. Thus i use name attribute in field on this code. i reviewed my previous lesson video, and yes it should not add name attribute. i will revise this code.